asm-utils/src/data_parser.rs

```diff
@@ -142,6 +160,18 @@ pub fn extract_data_objects<R: Register, F: FunctionOpKind>(
                         ));
                     }
                 }
+                (
+                    ".set",
+                    [Argument::Expression(Expression::Symbol(label)), Argument::Expression(Expression::Number(value))],
+                ) => {
```

Comment:
The other kind of `.set` directive is handled here. But I changed the comment to be less mysterious.